### PR TITLE
Fix some issues with stub

### PR DESCRIPTION
**Pull request by dktapps**
The first commit fixes an issue with method naming in Base64 stub (found by generating my own stub from the extension).

The second commit makes the stub valid PHP so that IDEs such as PhpStorm can use it for providing auto completion.

### CODE REVIEW

File: docs/Crypto.php [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <?php
+
+namespace Crypto{
```
Comment (bukka) [inline docs/Crypto.php]: Couldn't this just be `namespace Crypto;` or is it not working with IDE parser...?

Comment (bukka) [reply docs/Crypto.php]: just curios - not really an issue though... :) if it doesn't work, then it's fine - possibly there should be a space before `{` and the content indented but not really an issue either...

Comment (dktapps) [reply docs/Crypto.php]: You're right, I'll fix that.

**Comment by dktapps** [comment]
hey, any movement on this?

**Comment by bukka** [comment]
Sorry somehow forgot about this.

**Pull request merged**